00-HTML-CSS-basics/ex 1/1.6/index.html

```diff
@@ -0,0 +1,50 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="author" content="Maximiliano Tulian"/>
+    <meta name="description" content="Exercise 1.6" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Login form</title>
+</head>
+<body>
+    <section>
+        <form id="form-user" name="form-user">
+            <label for="txt-first-name">First Name</label>
+            <input type="text" name="txt-first-name" id="txt-first-name" pattern="[a-zA-Z]*" required autofocus/>
+            <br>
+
+            <label for="txt-last-name">Last Name</label>
+            <input type="text" name="txt-last-name" id="txt-last-name" pattern="[a-zA-Z]*" required/>
+            <br>
+
+            <label for="txt-email">Email</label>
+            <input type="email" name="txt-email" id="txt-email" placeholder="example@ex.com" required/>
+            <br>
+
+            <label for="txt-birthday">Birthday</label>
+            <input type="date" name="txt-birthday" id="txt-birthday" min="1915-01-01" required>
+            <br>
+            <!-- The min attribute of the input tag is not supported in Internet Explorer 9 and earlier versions, or in Firefox.-->
+            <!-- The date input type is not supported in all browsers-->
+
+            <label for="lst-sports">Select your favorite sport</label>
+            <select id="lst-sports" name="lst-sports">
+                <option value="1" selected>Calisthenics</option>
+                <option value="2">Footbal</option>
+                <option value="3">Voleyball</option>
+                <option value="4">Handball</option>
+            </select>
+            <br>
+
+            <label for="txt-bio">Write your biography</label><br>
+            <textarea id="txt-bio" name="txt-bio" placeholder="Please write a small biography of yourself" required></textarea>
+            <br>
+
+            <input type="submit" value="Sign Up">
```

Comment:
You should close this tag, it could be self closed, also all the `<br>` should be `<br/>`